Scenes/options.tscn

```diff
@@ -323,6 +341,8 @@ flat = false
 
 [connection signal="value_changed" from="PanelContainer/VBoxContainer/HBoxContainer 2/zoom_slider" to="." method="_on_zoom_slider_value_changed"]
 
+[connection signal="toggled" from="PanelContainer/VBoxContainer/enforce_8bit" to="." method="_on_enforce_8bit_btn_toggled"]
```

Comment:
Not an issue, I just want to admire how beautiful Godot looks in version control. If this was Unity, that change would avalanche through a 300MG binary file making merges impossible, unless you're using their propriety VCS, in which case everything would simply crash.

Reply:
I know right? Did Unity one single time, never again.
But I'm not really into game/desktop apps programming anyway, so it's not much of an issue for me.